torch/_torch_docs.py

```diff
@@ -3279,7 +3279,7 @@ def merge_dicts(*dicts):
 
 .. math::
     \text{{out}}_i = \text{{input}}_i - trunc(\frac{\text{{input}}_i}{\text{{other}}_}) * \text{{other}}_i
```

Comment:
`{\text{{other}}_})` looks like this is causing the issue. I think it should be `{\text{{other}}_i}`

Reply:
This fixed the issue. Thank you, @kshitij12345 !! 🚀